device-types/HP/HP-EliteDesk-800-G5-Mini.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: HP
+model: EliteDesk 800 G5 Mini
+slug: hp-elitedesk-800-g5-mini
+part_number: hp-elitedesk-800-g5-mini
+u_height: 1
+is_full_depth: false  # 15.75"
+weight: 2.31
+weight_unit: lb
+front_image: true
+rear_image: true
+comments: '[Datasheet](https://h20195.www2.hp.com/v2/getpdf.aspx/c06320288.pdf)'
+interfaces:
+  - name: ethernet-1/1
+    type: 1000base-t
+module-bays:
```

Comment:
As the power is integrated into the device it needs to be added as a `power-port` with `type: dc-terminal`.